src/api/server/server.js

```diff
@@ -4,11 +4,43 @@
 
 const _ = require('lodash');
 const common = require('../common');
+/* eslint-disable no-unused-vars */
+const Remote = require('../../core/remote').Remote;
+/* eslint-enable no-unused-vars */
 
 // If a ledger is not received in this time, consider the connection offline
 const CONNECTION_TIMEOUT = 1000 * 30;
 
-function isUpToDate(remote): boolean {
+type GetServerInfoResponse = {
+  buildVersion: string,
+  completeLedgers: string,
+  hostid: string,
+  ioLatencyMs: number,
+  load: {
+    job_types: Array<Object>,
```

Comment:
My bad. It is recursive, I just took that field from documentation and didn't changed.